README.md

```diff
@@ -11,165 +11,92 @@ decrypted only by the controller running in the target cluster and
 nobody else (not even the original author) is able to obtain the
 original Secret from the SealedSecret.
 
+<br>
+
 ## Installation
 
+There are two parts to installation:
+* [controller](#controller-installation) 
+* [kubeseal](#kubeseal-installation)
+
+**See [TPR->CRD migration guide](MIGRATION.md) if updating an existing Sealed Secrets installation from Kubernetes <= 1.7**
+
 See https://github.com/bitnami-labs/sealed-secrets/releases for the latest
 release.
 
-**See additional TPR->CRD migration section below if updating an
-existing Sealed Secrets installation from Kubernetes <= 1.7**
-
-```sh
-$ release=$(curl --silent "https://api.github.com/repos/bitnami-labs/sealed-secrets/releases/latest" | sed -n 's/.*"tag_name": *"\([^"]*\)".*/\1/p')
+<br>
 
-# Install client-side tool into /usr/local/bin/
-$ GOOS=$(go env GOOS)
-$ GOARCH=$(go env GOARCH)
-$ wget https://github.com/bitnami-labs/sealed-secrets/releases/download/$release/kubeseal-$GOOS-$GOARCH
-$ sudo install -m 755 kubeseal-$GOOS-$GOARCH /usr/local/bin/kubeseal
+## Controller Installation
 
-# Install SealedSecret TPR (for k8s < 1.7)
-$ kubectl create -f https://github.com/bitnami-labs/sealed-secrets/releases/download/$release/sealedsecret-tpr.yaml
+`controller.yaml` will install the `controller` into `kube-system` namespace, create a service account, necessary RBAC roles and a `SealedSecret` CRD.  
+After a few moments, the `controller` will start, generate a key pair, and be ready for operation.  If it does not, check the `controller` logs.
 
-# Install SealedSecret CRD (for k8s >= 1.7)
-$ kubectl create -f https://github.com/bitnami-labs/sealed-secrets/releases/download/$release/sealedsecret-crd.yaml
+The key certificate (public key portion) is used for sealing secrets, and will be printed to the `controller` log on startup.
 
+```sh
 # Note:  If installing on a GKE cluster, a ClusterRoleBinding may be needed to successfully deploy the controller in the final command.  Replace <your-email> with a valid email, and then deploy the cluster role binding:
 $ USER_EMAIL=<your-email>
 $ kubectl create clusterrolebinding $USER-cluster-admin-binding --clusterrole=cluster-admin --user=$USER_EMAIL
 
-# Install server-side controller into kube-system namespace (by default)
-$ kubectl create -f https://github.com/bitnami-labs/sealed-secrets/releases/download/$release/controller.yaml
+# For clusters with RBAC
+$ wget https://github.com/bitnami-labs/sealed-secrets/examples/controller.yaml
```

Comment:
Okay, sure will try that and verify flow with new tag in my fork.

Reply:
Generating and adding YAML's to archive with `builds.hooks.post` was straight forward with two exceptions:
1. `Makefile` target builds docker image as part of generating YAML, this duplicates goreleaser and could lead to different docker images, see [proposed change](https://github.com/bitnami-labs/sealed-secrets/pull/140/files#diff-b67911656ef5d18c4ae36cb6741b7965R61)
2. `Makefile` adds `+dirty` to `$(VERSION)` if repo dirty but `+` is invalid image tag character, see [proposed change](https://github.com/bitnami-labs/sealed-secrets/pull/140/files#diff-b67911656ef5d18c4ae36cb6741b7965R10)
I'm trying not to alter existing flow too much but there is overlapping functionality.

RE: Publishing to GitHub as part of Release:
Looks like the `Release` struct in [Goreleaser](https://github.com/goreleaser/goreleaser/blob/master/pkg/config/config.go#L159) doesn't have a `Files` var like some of the others.
I've asked on their Slack channel about it. Imagine it would take a bit of plumbing to wire it through if they are open to a PR. Maybe there's an alternate method?